darts/timeseries.py

```diff
@@ -1176,6 +1176,28 @@ def values(self, copy=True, sample=0) -> np.ndarray:
         else:
             return self._xa.values[:, :, sample]
 
+    def random_component_values(self, copy=True) -> np.array:
```

Comment:
What do you think about an additional optional to randomly sample a quantile per sample? 

I would assume this is similar to quantile regression based on stochastic input (without needing QuantileRegression as a likelihood)

Reply:
Oh, you mean returning the exact quantile instead of sampling a sample at random? That's an interesting idea :) But would require a bit of adaptation elsewhere as we would still need to ensure this is applied on the output targets only, and not e.g. on the inputs or covariates. Let's keep the idea though.